package.json

```diff
@@ -76,12 +76,13 @@
     "requestretry": "^1.12.0",
     "screwdriver-artifact-bookend": "^1.0.1",
     "screwdriver-build-bookend": "^2.0.1",
-    "screwdriver-config-parser": "^3.1.0",
-    "screwdriver-data-schema": "^16.0.1",
+    "screwdriver-config-parser": "^3.3.0",
+    "screwdriver-data-schema": "^16.1.1",
     "screwdriver-datastore-sequelize": "^2.0.0",
     "screwdriver-executor-docker": "^2.0.0",
     "screwdriver-executor-k8s": "^10.0.0",
     "screwdriver-models": "^21.2.0",
+    "screwdriver-notifications-email": "^1.0.3",
```

Comment:
❓ Super weird.

Just a wild guess, but it *might* correct itself after rebasing with master. If you want to pursue it on your local machine, I would suggest doing it on a separate branch so you don't lose all your hard work. Someone in the group has already lost their hard work to a git rebase gone wrong; don't be the next victim!